test/helpers/api_policy_helpers.rb

```diff
@@ -0,0 +1,118 @@
+module ApiPolicyHelpers
+  extend ActiveSupport::Concern
+
+  included do
+    attr_reader :rubygem, :owner
```

Comment:
Setting these attributes feels somewhat awkward for matchers that already require a given set of input. We can we look to pass in the `rubygem` and `owner` variables as well?

Reply:
Updated. I'm happier with it now. Thanks for the feedback.